engagement/jobs/WebsiteVendorMonitoring.groovy

```diff
@@ -46,8 +46,9 @@ class WebsiteVendorMonitoring {
             multiscm {
                 git {
                     remote {
-                        url('https://github.com/edx/prospectus.git')
+                        url('git@github.com:edx/prospectus.git')
                         branch('master')
+                        credentials(gitCredentialId)
```

Comment:
Is the credential above correct / where is it set?

Reply:
I have defined the var in edx-internal https://github.com/edx/edx-internal/pull/4420